### DISCUSSION TITLE
Looking for Solax Smart meter SDM230 protocol details

### DISCUSSION THREAD

**Discussion by straga** [arendst/Tasmota#15207] [Q&A] [answered]
Hello.
Maybe someone has information about modbus protocol for SDM230 with solax firmware.
More details.
https://github.com/syssi/esphome-modbus-solax-x1/issues/20

**Comment by sfromis** [comment, marked as answer]
The Tasmota [driver for SDM230](https://github.com/arendst/Tasmota/blob/development/tasmota/xnrg_21_sdm230.ino) does deal with the protocol, and includes some info.

**Comment by straga** [comment]
I need for SDM230 with solax firmware, that looks like a normal SDM230mbus mid.
